cli/flox/src/commands/environment.rs

```diff
@@ -891,6 +896,45 @@ impl Install {
         }
         Ok(())
     }
+
+    fn handle_error(
+        err: EnvironmentError2,
+        flox: &Flox,
+        environment: &dyn Environment,
+        packages: &[PackageToInstall],
+    ) -> anyhow::Error {
+        debug!("install error: {:?}", err);
+
+        match err {
+            EnvironmentError2::Core(CoreEnvironmentError::LockedManifest(
+                LockedManifestError::LockManifest(
+                    flox_rust_sdk::models::pkgdb::CallPkgDbError::PkgDbError(pkgdberr),
+                ),
+            )) if pkgdberr.exit_code == 120 => 'error: {
+                let paths = packages.iter().map(|p| p.path.clone()).join(", ");
+                let head = format!("❌  could not install {paths}");
```

Comment:
nit: I think it would be better to print `could not install` and then have the spinner pop up, rather than the first message be `looking for suggestions` and then printing `could not install` after that finishes

Reply:
That would be a bit weird since then `Error` would be the suggestion, i.e. we would print an error outside of the error handling..
I'd like to keeo it self contained for now if possible.

Reply:
Which do you think is a better user experience?

Reply:
We can make amend the spinner to say "Could not find a package for {path}, searching for suggestions"...
I think that's both clear and does not fork the error handling

Reply:
👍 great

Reply:
added the message, needs new approval